WalletWasabi.Gui/ViewModels/StatusBarViewModel.cs

```diff
@@ -253,6 +257,36 @@ public void Initialize(NodesCollection nodes, WasabiSynchronizer synchronizer)
 
 					UpdateAvailable = !x.ClientUpToDate;
 					CriticalUpdateAvailable = !x.BackendCompatible;
+
+					try
+					{
+						if (Global.LegalDocuments is null || Global.LegalDocuments.Version != x.LegalDocumentsVersion)
+						{
+							try
+							{
+								(LegalDocuments legalDocuments, string content) legalResp;
+								if (Global.Config.UseTor)
+								{
+									legalResp = await LegalDocuments.FetchLatestAsync(Global.DataDir, () => Global.Config.GetCurrentBackendUri(), Global.Config.TorSocks5EndPoint, CancellationToken.None);
+								}
+								else
+								{
+									legalResp = await LegalDocuments.FetchLatestAsync(Global.DataDir, () => Global.Config.GetFallbackBackendUri(), null, CancellationToken.None);
+								}
```

Comment:
Can be converted to conditional expression:
```
(LegalDocuments legalDocuments, string content) legalResp = Global.Config.UseTor
	? await LegalDocuments.FetchLatestAsync(Global.DataDir, () => Global.Config.GetCurrentBackendUri(), Global.Config.TorSocks5EndPoint, CancellationToken.None)
	: await LegalDocuments.FetchLatestAsync(Global.DataDir, () => Global.Config.GetFallbackBackendUri(), null, CancellationToken.None);
```

Reply:
Looks ugly. But I'm ok with it if others want that, too.

Reply:
We can de-uglify it a bit:

```c#
var (backendUrl, torSocks5Endpoint) = Global.Config.UseTor
             ? (Global.Config.GetCurrentBackendUri(), Global.Config.TorSocks5EndPoint)  
             : (Global.Config.GetFallbackBackendUri(), null);
var (legalDocuments, content) legalResp = await LegalDocuments.FetchLatestAsync(Global.DataDir, () => backendUrl, torSocks5Endpoint, CancellationToken.None);
```

Reply:
Meh. Closing this. This conversation is pointless and can drag on taking our time.